### DISCUSSION TITLE
difference between opendtu-generic and opendtu-generic.factory bins

### DISCUSSION THREAD

**Discussion by markusruebe** [tbnobody/OpenDTU#1027] [Q&A] [answered]
Dear all,
what is the difference between the generic and the generic factory bin file?
I am always using the generic.bin file, but just was wondering what the factory version is used for. 

thanks and greetings
markus

**Comment by Solarteur** [comment]
Compare #1027

**Comment by Elkomaster91** [comment, marked as answer]
The factory is for new installation and the other for updates
